tests/unit/policyEvaluator.js

```diff
@@ -332,6 +332,24 @@ describe('policyEvaluator', () => {
                         check(requestContext, rcModifiers, policy, 'Allow');
                     });
 
+            it('should allow access for StringNotLike condition with ' +
+            'variables and wildcards: without any prefix in the query',
```

Comment:
it should allow access for StringNotLike condition if condition parameter is completely missing from request